myst_nb/parser.py

```diff
@@ -124,14 +125,40 @@ def parse(self, inputstring, document):
         except AttributeError:
             pass
 
-        # Write the notebook's output to disk
+        # Remove all the mime prefixes from "glue" step.
```

Comment:
maybe add a configuration option to turn this functionality on/off

Reply:
This is needed in order to ensure that the files actually get written to disk. The jupyter-sphinx writer step only writes image files that it has prefixes for. So if we have the `nteract/application` stuff in there it'll skip them.

Another option would be to try and upstream some logic to `jupyter-sphinx` that'll check for a prefix...

Reply:
I meant to turn off the entire glue functionality, but obviously anythingupstreamed tojupyter-sphinx is probably good

Reply:
@chrisjsewell ah that's a good point, agreed

Reply:
@akhmerov would you be +1 on an addition to the "write_outputs" function that would allow us to add a "prefix" to any outputs that it would also search for? That way `image/png` would work as well as `myprefix/image/png`.